src/rhsmlib/services/refresh.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+# Red Hat trademarks are not licensed under GPLv2. No permission is
+# granted to use or replicate Red Hat trademarks that are incorporated
+# in this software or its documentation.
+
+"""
+This module provides service for refreshing entitlement certificates
+"""
+
+
+import logging
+
+import subscription_manager.injection as inj
+from subscription_manager.entcertlib import EntCertActionInvoker
+
+log = logging.getLogger(__name__)
+
+
+class Refresh(object):
+    """
+    Class used for refreshing entitlement certificates
+    """
+
+    def __init__(self, cp=None, ent_cert_lib=None):
```

Comment:
both the new callers of this pass both `cp` and `ent_cert_lib`; to make things easier for now, should we avoid the default arguments & code for the `None` cases?

Reply:
Yes, new callers pass something, but future callers needn't. Default arguments makes usage of this class easier.